### PR TITLE
HeliumServer for GoogleEngine deploy

### PR DESCRIPTION
**Pull request by Burbaka**
@roman-mazur @Vandalko please review

**Development:**
To run and test local server:
1. Uncomment endpoint in api specification file.
2. Run gradle command with `./gradlew appengineRun`
3. Run Helium tests with `./gradlew checkApiBeh` task

**Deploy:**
To upload and deploy new server intance on CloudEngine run `./gradlew appengineUpdate` task

### CODE REVIEW

**Comment by roman-mazur** [comment]
:+1: We need to configure PR builds. @Burbaka Would you add travis integration?

**Comment by Burbaka** [comment]
@roman-mazur I may try)

**Comment by roman-mazur** [comment]
We actually do not need retrofit interface generation. It's the server side only (for now).

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - JAVA_OPTS=-XX:MaxPermSize=1024m
+
+git:
+  depth: 5
+
+install:
+- ./gradlew -q -s clean assemble
+
+script:
+- ./gradlew -s --continue check checkApiBehaviour
```
Comment (roman-mazur) [inline .travis.yml]: you do not start the local server, how will you check the api behaviour? ;)

**Comment by Burbaka** [comment]
@roman-mazur Travis CI was added

**Comment by Burbaka** [comment]
By default when you run `gw checkAPiBehaviour` it executes on localhost:8080 .
To use CloudEngine endpoint you should pass a parameter `endpoint`. 
For example, `gw chApiBeh -Pendpoint=http://.....`

**Pull request merged**